src/overlays/actors/ovl_Obj_Syokudai/z_obj_syokudai.c

```diff
@@ -82,7 +79,8 @@ static InitChainEntry sInitChain[] = {
 
 static u8 sColTypes[] = { COLTYPE_METAL, COLTYPE_WOOD, COLTYPE_WOOD };
 
-static Gfx* sDLists[] = { D_060003A0, D_06000B90, D_06000870 };
+static Gfx* sDLists[] = { sObjectSyokudaiTypeSwitchCausesFlameDL, sObjectSyokudaiTypeFlameCausesSwitchDL,
+                          sObjectSyokudaiTypeNoSwitchDL };
```

Comment:
```suggestion
static Gfx* sDLists[] = { sObjectSyokudaiTypeSwitchCausesFlameDL, sObjectSyokudaiTypeFlameCausesSwitchDL,
                          sObjectSyokudaiTypeNoSwitchDL, };
```
With these named it would probably look better formatted with a trailing comma.